README.md

```diff
@@ -55,6 +55,7 @@ The following table describes the samples and root files in this repository:
 | File/folder | Description | Language/Platform |
 |-------------|-------------|-------------------|
 | [`clients\csharp-wpf`](https://github.com/Azure-Samples/Cognitive-Services-Voice-Assistant/tree/master/clients/csharp-wpf) |  Windows voice assistant client sample. Generic Windows tool to manually test your bot or Custom Commands application | C#, Windows Presentation Foundation (WPF) |
+| [`clients\csharp-uwp`](https://github.com/Azure-Samples/Cognitive-Services-Voice-Assistant/tree/master/clients/csharp-uwp) |  UWP Voice Assistant Sample. Generic UWP voice agent demonstrating voice activation on Windows and Direct Line Speech. | C#, Universal Windows Platform (UWP) |

```

Comment:
Feel free to put this as the first row in the table, since it's super awesome cool. 
To better align with the other table rows, I would write in the middle column:
"UWP voice assistant client sample. An example UWP app that uses Window's new voice activation APIs, together with the Voice Assistant platform, to connect to your bot or Custom Commands application"